Knoema.Client/Client.cs

```diff
@@ -117,10 +120,27 @@ private HttpClient GetApiClient(int timeout = HttpClientTimeout)
                 builder.Query = "access_token=" + _token;
 
             var postResponse = await GetApiClient(timeout).PostAsync(builder.Uri, content);
+            EnsureSuccessApiCall(postResponse);
             var readString = await postResponse.Content.ReadAsStringAsync();
             return JsonConvert.DeserializeObject<T>(readString);
         }
 
+        private static void EnsureSuccessApiCall(HttpResponseMessage postResponse)
+        {
+            if (!postResponse.IsSuccessStatusCode)
+            {
+                var error = "";
+                if (postResponse.Content != null)
+                {
+                    error = postResponse.Content.ReadAsStringAsync().Result;
+                    error = Regex.Replace(error, "<style>(.|\n)+?</style>|<[^>]+>", String.Empty, RegexOptions.Multiline);
+                    error = Regex.Replace(error, @"\r\n\s*\r\n", "\r\n").Trim();
+                }
+                throw new Exception(
+                    String.Format("Remote server returned error {0}\r\n\r\n{1}", (int)postResponse.StatusCode, error));
```

Comment:
environment.newline instead of \r\n

Reply:
It does not depend on environment where Knoema.Client is used. It's server-generated error message.

Reply:
Changed.